buildSrc/src/main/kotlin/Dependencies.kt

```diff
@@ -9,5 +9,6 @@ object Dependencies {
 
     object Kotlinx {
         const val coroutinesCore = "org.jetbrains.kotlinx:kotlinx-coroutines-core:1.6.0-native-mt"
+        const val coroutinesTest = "org.jetbrains.kotlinx:kotlinx-coroutines-test:1.6.0-native-mt"
```

Comment:
We can use a private const `coroutinesVersion` here to make this easier to maintain